### DISCUSSION TITLE
Is it possible to change the commit message after the push?

### DISCUSSION THREAD

**Discussion by dariohsb** [community/community#86595] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I would like to know if after push the commit changes, i can like "rename" the commit message.
Thank you.

**Comment by Mahitej28** [comment]


**Reply by dariohsb** [reply]
Thank you mate!

**Reply by queenofcorgis** [reply]
@Mahitej28 

We’ve clarified our stance on using generative AI tools like ChatGPT within our Community via [this announcement](https://github.com/orgs/community/discussions/69006). Please review the guidelines to ensure your post meets them as failure to adhere to those rules can result in action taken by our moderator team.  You can read our updated [Code of Conduct](https://github.com/community/community/blob/baa5cd9020ef3f2bbf86dca1191d9c90f9b0822d/CODE_OF_CONDUCT.md#reasonable-use-of-ai-generated-content) and the [announcement](https://github.com/orgs/community/discussions/69006) for more details. Thank you for helping us maintain an authentic and beneficial space for everyone.

**Reply by dariohsb** [reply]
@queenofcorgis was that reply made by AI?

**Reply by Mahitej28** [reply, marked as answer]
Sure would keep in mind although It wasn't entirely AI-generated though
